### PR TITLE
feature: add tooltip, copy address, and see in block explorer

### PR DESCRIPTION
**Pull request by farrellh1**
# Description

Add tooltip and copy address feature

https://www.loom.com/share/e2898110f0ab44889c2a8b59cf47983e?sid=8630765f-1fc4-4576-a184-aca1bac9a086

Fixes # (#243)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

* **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)

### CODE REVIEW

**Comment by farrellh1** [comment]
there is a problem(see picture _before_) with tooltip and modal from daisyui, its because of the `overflow-y-auto`, when I changed it to `overflow-y-visible`, there is a side effect (see picture side-effect). wdyt? is there any better solution?

before:
image 

after: 
image
with side effect:
image

File: app/src/components/forms/__tests__/EditUserForm.spec.ts
Comment (hermannleboss) [inline app/src/components/forms/__tests__/EditUserForm.spec.ts:1]: Can you add more test to make this component reach the 100% of coverage ?

**Review by hermannleboss — COMMENTED** [review]
LGTM
Can try to remove items that are defined but are never used ?

The are in the warning when you run npm run type-check

**Comment by farrellh1** [comment]
btw, should i changed the UI of the members first to be like the one you suggestion @hermannleboss or just use this one?

**Pull request merged**